internal/installation/receipt/receipt.go

```diff
@@ -41,3 +42,19 @@ func Store(plugin index.Plugin, dest string) error {
 func Load(path string) (index.Receipt, error) {
 	return indexscanner.ReadReceiptFromFile(path)
 }
+
+// New returns a new receipt with the given plugin and index name.
+func New(plugin index.Plugin, indexName string) index.Receipt {
+	name := indexName
+	if name == "" {
+		name = constants.DefaultIndexName
+	}
```

Comment:
under what condition we specify "" ?

Reply:
You're right, I forgot the `CanonicalPluginName` function returns the default index already. I can remove this.

Reply:
Even without that, this should not happen. We'll always have an index name in pkg/installation.

`CanonicalPluginName` should be used only for turning user input (e.g. install args) to full plugin names.

Reply:
It sounds like you are talking about the same thing. Anyway, everybody agrees that the check `name == ""` is unnecessary.